frontend/src/components/RuleEditForm.vue

```diff
@@ -102,9 +104,16 @@ const formReady = computed(() => generationRulesCount.value > 0);
     :actions="false"
     :value="props.rule"
   >
-    <CRow class="mb-2">
-      <CCol xs="auto" class="me-auto">
-        <h4>
+    <CRow class="mb-2 d-flex">
+      <CCol xs="auto" class="flex-grow-1 d-flex align-items-center h3">
+        <FormKit
+          type="checkbox"
+          name="disabled"
+          :v-model="disabledValue"
```

Comment:
You don't actually need this line. FormKit will use the field name to match the incoming and outgoing data. (you therefore also don't need to define `disabledValue` at all)